src/__experimental__/components/search/search.stories.js

```diff
@@ -1,11 +1,11 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Something in this changes has caused some styling regressions in the test story

![image](https://user-images.githubusercontent.com/44157880/96995877-cc921b80-1526-11eb-94dd-c8ea72e8c7f4.png)
![image](https://user-images.githubusercontent.com/44157880/96995891-d3209300-1526-11eb-9cb0-07279232ff0b.png)

Reply:
Great point!